timemachine/cpp/src/harmonic_angle.cu

```diff
@@ -37,30 +27,9 @@ HarmonicAngle<RealType>::HarmonicAngle(
 
     gpuErrchk(cudaMalloc(&d_angle_idxs_, A_ * 3 * sizeof(*d_angle_idxs_)));
     gpuErrchk(cudaMemcpy(d_angle_idxs_, &angle_idxs[0], A_ * 3 * sizeof(*d_angle_idxs_), cudaMemcpyHostToDevice));
-
-    gpuErrchk(cudaMalloc(&d_lambda_mult_, A_ * sizeof(*d_lambda_mult_)));
-    gpuErrchk(cudaMalloc(&d_lambda_offset_, A_ * sizeof(*d_lambda_offset_)));
-
-    if (lambda_mult.size() > 0) {
-        gpuErrchk(cudaMemcpy(d_lambda_mult_, &lambda_mult[0], A_ * sizeof(*d_lambda_mult_), cudaMemcpyHostToDevice));
-    } else {
-        initializeArray(A_, d_lambda_mult_, 0);
-    }
-
-    if (lambda_offset.size() > 0) {
-        gpuErrchk(
-            cudaMemcpy(d_lambda_offset_, &lambda_offset[0], A_ * sizeof(*d_lambda_offset_), cudaMemcpyHostToDevice));
-    } else {
-        // can't memset this
-        initializeArray(A_, d_lambda_offset_, 1);
```

Comment:
nit: it looks like `initializeArray` is now unused; do we want to remove it?

Reply:
Good catch -- `initializeArray` is now unused.

Considered removal in https://github.com/proteneer/timemachine/pull/891/commits/89be7845558a2ec4f70490c6d5fb3e723b88017c, but reverted in https://github.com/proteneer/timemachine/pull/891/commits/c915e9bbc378f91062b01c949d294a698a3b0ec5, since the kernel `k_initialize_array` it wraps is used in the neighborlist.